src/Features/Core/Portable/EmbeddedLanguages/Json/LanguageServices/AbstractJsonDetectionCodeFixProvider.cs

```diff
@@ -69,7 +69,7 @@ public void Fix(SyntaxEditor editor, Diagnostic diagnostic, CancellationToken ca
         private class MyCodeAction : CodeAction.DocumentChangeAction
         {
             public MyCodeAction(Func<CancellationToken, Task<Document>> createChangedDocument)
-                : base(FeaturesResources.Enable_JSON_editor_features, createChangedDocument, nameof(FeaturesResources.Enable_JSON_editor_features))
+                : base(FeaturesResources.Enable_all_JSON_editor_features, createChangedDocument, nameof(FeaturesResources.Enable_all_JSON_editor_features))
```

Comment:
changed this string because `Enable_JSON_editor_features` doesn't make a huge amount of sense as some json features are already enabled (classification/bracehighlighting).  This enables the rest (currently 'diagnostics').  We could rename this to Enable_JSON_diagnostics if we think that would be clearer.